app/views/portals/thread-id.vue

```diff
@@ -1,181 +1,135 @@
-            <a :href="forumUrl.replace(/^\/+/, '')" class="btn btn-light" title="Back to forum">
+            <a :href="forumUrl.replace(/^\/+/, '')" class="btn btn-light" :title="t('backToForum')">
               <i class="fa fa-chevron-left" aria-hidden="true"></i>
             </a>
           </div>
           <div class="col align-self-center">
             <b>{{ lstring(thread.subject) }}</b>
-            <div v-if="forum && forum.isClosed"><em>{{ $t('forumIsClosedForComments') }}</em></div>  
-            <div v-else-if="thread.isClosed"><em>{{ $t('threadIsClosedForComments') }}</em></div>  
+            <div v-if="forum?.isClosed"><em>{{ t('forumIsClosedForComments') }}</em></div>
+            <div v-else-if="thread?.isClosed"><em>{{ t('threadIsClosedForComments') }}</em></div>
           </div>
-
           <div class="col-auto align-self-center">
-
             <button v-if="isOpen && subscription" :disabled="subscribing"
-              class="btn btn-sm" :class="{ 'btn-outline-dark':!subscription.watching, 'btn-dark': subscription.watching }" type="button"
+              class="btn btn-sm" :class="{ 'btn-outline-dark': !subscription.watching, 'btn-dark': subscription.watching }" type="button"
               @click="toggleSubscription()">
-              <span v-if="subscription.watching"><i class="fa fa-envelope-o"></i> {{ $t('buttonUnsubscribe') }} </span>
-              <span v-else><i class="fa fa-envelope-o"></i> {{ $t('buttonSubscribe') }} </span>
+              <span v-if="subscription.watching"><i class="fa fa-envelope-o"></i> {{ t('buttonUnsubscribe') }} </span>
+              <span v-else><i class="fa fa-envelope-o"></i> {{ t('buttonSubscribe') }} </span>
               <i v-if="subscribing" class="fa fa-cog fa-spin"></i>
             </button>
-
           </div>
         </div>
       </div>
-
-      <h1>{{ lstring(thread.subject) }}</h1> 
-
-      <post :post="thread" class="mb-2" @refresh="refresh()">
+      <h1>{{ lstring(thread.subject) }}</h1>
+      <post :post="thread" class="mb-2" @refresh="load()">
         <template v-slot:showReplies="{ replies }">
-            <em>
-              <a class="anchor-margin" name="replies"></a>
-              <span v-if="replies == 0"> {{ $t('linkNoReplies', { count: replies }) }}</span>
-              <span v-if="replies == 1"><i class="fa fa-comment"></i> {{ $t('linkOneReply', { count: replies }) }} </span>
-              <span v-if="replies  > 1"><i class="fa fa-comments"></i> {{ $t('linkXReplies', { count: replies }) }} </span>
-            </em>
+          <em>
+            <a class="anchor-margin" name="replies"></a>
+            <span v-if="replies == 0">{{ t('linkXReplies', { count: replies }) }}</span>
+            <span v-if="replies == 1"><i class="fa fa-comment"></i> {{ t('linkXReplies', { count: replies }) }}</span>
+            <span v-if="replies  > 1"><i class="fa fa-comments"></i> {{ t('linkXReplies', { count: replies }) }}</span>
```

Comment:
can we make it on line with `v-if`? I think we can drop the `comment` vs `comments`.
or we keep it with `:class`
```suggestion
            <span>
                <i v-if="replies>0" class="fa" : class="{ 'fa-comment':replies==1, 'fa-comments':replies>1 }"></i>
                {{ t('linkXReplies', { count: replies }) }}
            </span>
```